dynawo/sources/Modeler/Common/DYNSubModel.h

```diff
@@ -730,6 +730,14 @@ class SubModel {
    */
   void addMessage(const std::string& message);
 
+  /**
+   * @brief getter to tell whether or not simulation generates a timeline
+   * @return true when simulation generates a file resuming the timeline; false when timeline has not been asked from jobs file
```

Comment:
"false when timeline has not been asked from jobs file" => "false otherwise"

Reply:
ok